src/IonLocalSymbolTable.ts

```diff
@@ -37,8 +37,7 @@ export class LocalSymbolTable  {
   }
 
   getSymbolId(symbol_: string) : number {
-    return this._import.getSymbolId(symbol_)
-      || this.index[symbol_];
+    return this._import.getSymbolId(symbol_) || this.index[symbol_];
```

Comment:
Can an LST have multiple imports?

Reply:
You have to chain their creation like: 
```
    var symbolTable = new ion.SharedSymbolTable('foo', 1, ['a', 'b', 'c']);
    var parent = new ion.Import(null, symbolTable, 1);
    var child = new ion.Import(parent, symbolTable);
```

Reply:
theres a few tests in tests/unit that go over symbol tables and imports.

Reply:
also that makes a lookup follow child to parent until it is able to resolve the sid

Reply:
This feels kind of icky. Do you think it's simpler than just having a list of Imports?

Reply:
Is the test coverage adequate? Given that ion-tests doesn't fully exercise shared symbol table cases (yet).